src/EditorFeatures/Core/Suggestions/SuggestedActionsSource.cs

```diff
@@ -104,13 +105,29 @@ public bool TryGetTelemetryId(out Guid telemetryId)
                 }
             }
 
+            private bool IsSolutionLoadComplete()
+            {
+                if (_workspace != null)
+                {
+                    var uiService = _workspace.Services.GetService<IHostContextService>();
+                    return uiService.SolutionExistsAndFullyLoaded;
```

Comment:
I was hoping it was that the solution existed and was fully loaded.   But my understanding now is that it doesn't mean anything anymore...